searchlib/src/vespa/searchlib/features/utils.cpp

```diff
@@ -1,28 +1,43 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
 #include "utils.hpp"
+#include <charconv>
 
 namespace search::features::util {
 
+template <typename T>
+T strToInt(vespalib::stringref str)
+{
+    T retval = 0;
+    if ((str.size() > 2) && (str[0] == '0') && ((str[0] | 0x20) == 'x')) {
+        std::from_chars(str.data(), str.data()+str.size(), retval, 16);
```

Comment:
According to https://en.cppreference.com/w/cpp/utility/from_chars `from_chars` does not recognize the 0x prefix, so we should skip it in offset and length. If we do not have one already, a unit test for this would be nice.

Reply:
Unit test added and code fixed.